.packit.yaml

```diff
@@ -1,14 +1,14 @@
-specfile_path: rpm-head-signing.spec
+specfile_path: python-rpm-head-signing.spec
 
-synced_files:
-    - rpm-head-signing.spec
+files_to_sync:
+    - python-rpm-head-signing.spec
     - .packit.yaml
+create_pr: false
 sync_changelog: true
 
 upstream_package_name: rpm-head-signing
-upstream_tag_template: v{version}
 upstream_project_url: https://github.com/fedora-iot/rpm-head-signing
-downstream_package_name: rpm-head-signing
+downstream_package_name: python-rpm-head-signing
```

Comment:
That is not correct, the downstream is as it is there.

Reply:
Sorry, I didn't realize the script was changing more things than needed.  I will investigate why. The only needed change is renaming `synced_files` to `files_to_sync`.